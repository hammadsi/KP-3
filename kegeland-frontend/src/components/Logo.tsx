import { Heading } from '@chakra-ui/react';

const Logo: React.FC = () => {
  return (
    <Heading
      as="h1"
      fontSize={26}
      letterSpacing={3}
      textTransform="uppercase"
      fontWeight="thin"
      color="primary.700"
    >
      <span style={{ fontWeight: '600' }}>NTNU</span> Physiotherapy
    </Heading>
  );
};

export default Logo;
