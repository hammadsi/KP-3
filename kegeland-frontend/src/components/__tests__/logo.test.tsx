import renderer from 'react-test-renderer';

import Logo from '../Logo';

describe('Test logo', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Logo />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
