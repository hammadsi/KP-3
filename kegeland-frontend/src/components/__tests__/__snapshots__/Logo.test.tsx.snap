// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test logo renders correctly 1`] = `
<h1
  className="chakra-heading css-1hss139"
>
  <span
    style={
      Object {
        "fontWeight": "600",
      }
    }
  >
    NTNU
  </span>
   Physiotherapy
</h1>
`;
